index.js

```diff
@@ -1,62 +1,53 @@
 'use strict'
 
-const figgyPudding = require('figgy-pudding')
 const getStream = require('get-stream')
 const npmFetch = require('npm-registry-fetch')
 
-const SearchOpts = figgyPudding({
-  detailed: { default: false },
-  limit: { default: 20 },
-  from: { default: 0 },
-  quality: { default: 0.65 },
-  popularity: { default: 0.98 },
-  maintenance: { default: 0.5 },
-  sortBy: {}
-})
-
 module.exports = search
 function search (query, opts) {
   return getStream.array(search.stream(query, opts))
```

Comment:
Probalby something like: `return search.stream(query, opts).collect()` (assuming `search.stream()` returns a minipass.)

Reply:
I see, yeah, it should be a minipass object stream once npm-registry-stream is updated to latest.